index.html

```diff
@@ -0,0 +1,121 @@
+                    <h3>"Hello Everyone, Let's save animals"</h3>
+                    <h1>SAVE ANIMALS COMMUNITY 2022</h1>
+                </div>
+                <P>Welcome to the anual gathering made by the community of "Save Animals". this years event is going to
+                    be on a large scale for all communities to participate.</P>
+                <div class="home-date">
+                    <h2>2022.10.15(THU) ~ 16(FRI)</h2>
+                    <h4>@ Music, meals, drinks and more</h4>
+                </div>
+            </div>
+        </section>
+        <section class="about-section" id="program">
+            <div class="about-container">
+                <div class="about-title">
+                    <h3>Main Program</h3>
+                    <div class="divider"></div>
+                </div>
+            </div>
+            <div class="about-cards">
+                <article class="card-box">
+                    <i class="fa-solid fa-comment fa-4x"></i>
+                    <h4>Lecture</h4>
+                    <p>A brief speak reminding the community why is so important to care and preserve animals around the
+                        world
+                    </p>
+                </article>
+                <article class="card-box">
+                    <i class="fa-regular fa-face-smile-beam fa-4x"></i>
+                    <h4>Exhibition</h4>
+                    <p>Kids from the elementary school are going to present a beautiful exhibition of cards and banners
+                        about sheltered animals.
+                    </p>
+                </article>
+                <article class="card-box">
+                    <i class="fa-solid fa-guitar fa-4x"></i>
+                    <h4>Concerts</h4>
+                    <p>Two popular pop bands are going to preform their most beloved songs followed by the best singer
+                        in town.
+                    </p>
+                </article>
+                <article class="card-box">
+                    <i class="fa-solid fa-paw fa-4x"></i>
+                    <h4>Adopt</h4>
+                    <p>We will be giving for adoption a variaty of rescued cats and dogs for anyone interested
+                        (restrictions may apply)
+                    </p>
+                </article>
+                <article class="card-box">
+                    <i class="fa-solid fa-utensils fa-4x"></i>
+                    <h4>Buffet</h4>
+                    <p>A large buffet for all the guests with a lot of vegan choices (included in payment)
+                    </p>
+                </article>
+            </div>
+            <button class="about-btn">Join our community</button>
+        </section>
+        <section class="featured-section" id="featured">
+            <h3>Comments from last year's event</h3>
+            <span class="divider"></span>
+            <div class="featured-cards" id="feature">
+            </div>
+        </section>
```

Comment:
## REQUIRED

- [ ] Please change the div tags to article tags just as the [previous reviewer](https://github.com/fasaldana/capstone_project/pull/1#discussion_r825199922) requested, this will make this section more semantic and easily readable by screen readers 🤗  


## OPTIONAL
Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![commu2](https://user-images.githubusercontent.com/69638013/158007867-fcefb0a3-ee92-4db0-948c-55305f50c40a.png)  |  ![commu](https://user-images.githubusercontent.com/69638013/158007861-92cddb60-2008-480d-94fa-5ee2ea018b48.png)

- [ ] It would be better for the button text "Join our community" to be on one line 
- [ ]  Kindly make the "featured-section" text look like the original, for example, the h4's  text was italicized also add the appropriate spacing for the main heading of this section 👍🏿 
- Good job adding functionality to the more and less buttons 💪🏿